libs/ledger-live-common/src/e2e/enum/Currency.ts

```diff
@@ -53,7 +53,7 @@ export class Currency {
     "bsc",
     AppInfos.BINANCE_SMART_CHAIN,
   );
-  static readonly TON = new Currency("Ton", "TON", "ton", AppInfos.TON);
+  static readonly TON = new Currency("TON", "TON", "ton", AppInfos.TON);
```

Comment:
not sure about this 🤔

Reply:
Actually, that doesn't affect [LLD](https://ledger-live.allure.green.ledgerlabs.net/allure/reports/0279358c-2f56-46d6-b7a1-2209ba3071d6/#)